### PR TITLE
Fix math formula rendering in Poisson Distribution docs.

### PR DESCRIPTION
**Pull request by ghost**
Trivial patch, tested with `cd docs && make html`, thanks for review @rachtsingh ! :)

Before patching:
screen shot 2018-05-08 at 1 07 10 am

After patching:
screen shot 2018-05-08 at 1 02 16 am

### CODE REVIEW

File: torch/distributions/poisson.py [hunk 1]
```diff
@@ -11,7 +11,9 @@ class Poisson(ExponentialFamily):
     Creates a Poisson distribution parameterized by `rate`, the rate parameter.
 
     Samples are nonnegative integers, with a pmf given by
-    $rate^k e^{-rate}/k!$
+
+    .. math::
+      rate^k e^{-rate}/k!
```
Comment (apaszke) [inline torch/distributions/poisson.py]: Can you please make it `\mathrm{rate}`? Right now it renders the word as if it was a sequence of 4 variables to multiply

Comment (apaszke) [reply torch/distributions/poisson.py]: Also, can you make it `\frac{e^{-\mathrm{rate}}}{k!}` to make the fraction render nicely?

**Comment by fracting** [comment]
@apaszke good point! Thanks for the review. I updated the pull request.

new rendering result:

screen shot 2018-05-08 at 2 31 26 am

**Review by apaszke — APPROVED** [review]

**Comment by apaszke** [comment]
Thanks @fracting!

**Pull request merged**